build/build.go

```diff
@@ -605,13 +635,12 @@ func (db *DeploymentBuilder) buildPlatformNetworks(d *Deployment) error {
 			// We are passing pointer to PlatformNetwork resource (net), hence, calling filterPlatformNetworks
 			// before or after append wouldn't make any difference and either way it would work.
 			d.PlatformNetworks = append(d.PlatformNetworks, net)
-			err = db.filterPlatformNetworks(net, d)
+			err = db.filterPlatformNetworks(net, addrpool, d)
```

Comment:
I think `addrpool` is unnecessary argument, please remove if it's not used.

Reply:
Removed addrpool